extensions/interactions/LogicProof/static/js/generatedParser.ts

```diff
@@ -720,11 +720,11 @@ var logicProofParser = (function(){
           pos = pos1;
         }
         if (result0 !== null) {
-          result0 = (function(offset, name, arguments) {
+          result0 = (function(offset, name, args) {
```

Comment:
These changes are needed to fix the TypeScript errors?

Reply:
If so could you also add information about this change into the README?

Reply:
> These changes are needed to fix the TypeScript errors?

Yes they are. A corresponding change has been made to input_to_PEG.txt.